src/docs/antora/modules/ROOT/pages/events.adoc

```diff
@@ -142,6 +146,17 @@ That API is available through the `spring-modulith-events-api` artifact, that yo
 </dependency>
 ----
 
+Gradle::
++
+[source, subs="+attributes", role="secondary"]
+----
+dependencies {
+  compile 'org.springframework.modulith:spring-modulith-events-api:{projectVersion}'
```

Comment:
According to the [reference docs](https://docs.gradle.org/5.3.1/userguide/java_plugin.html#sec:java_plugin_and_dependency_management), `implementation` is preferred here.